source/configuration_manual/stats/index.rst

```diff
@@ -52,6 +52,21 @@ generation based on event fields' values.  Each field listed in the
 generated metrics are indistinguishable from those statically defined
 in the config file.
 
+Dovecot supports a number of aggregate functions that can be used to
+quantize a field's value before it is used to generate a metric.
+
+The format is always the same: the field name, a colon, the aggregation
+function name, and an optional colon followed by parameters to the
```

Comment:
if both the colon and parameters are optional, would it be clearer to say "and optionally a colon followed by.."?

Looking at e.g. exponential, it seems multiple arguments are also separated by colons. Is that necessary to mention here already?

Reply:
I was trying to avoid saying that the parameters are colon separated, since (in theory) each function could do whatever it wants to... but it is probably simpler to say that they are all colon delimited and deal with that not being true when we actually come across a usecase that needs that.